web/template/search.gohtml

```diff
@@ -88,7 +88,7 @@
                                 </td>
                                 <td class="govuk-table__cell">{{ formatDate .DateOfBirth }}</td>
                                 <td class="govuk-table__cell">{{ .AddressSummary }}</td>
-                                <td class="govuk-table__cell">
+                                <td class="govuk-table__cell table-cell-nowrap">
                                     {{ range $case := .Cases }}
```

Comment:
Do you have a screenshot showing what this does to the layout? I'm not sure whether we should be using linebreaks to space things out. Could we be using paragraphs instead? That should remove the need for nowrap and white space tweaks.

Reply:
You should indeed avoid using line breaks for formatting and wrap in appropriate element instead. Might be a list in this case if it's a list of cases.

Reply:
I'll make a separate ticket for that just because it passed testing earlier and it was a nightmare trying to get a dev environment up